statusHandler/presenter/common.go

```diff
@@ -30,3 +35,45 @@ func (psh *PresenterStatusHandler) getFromCacheAsString(metric string) string {
 
 	return valStr
 }
+
+func plural(count int, singular string) (result string) {
+	if (count == 1) || (count == 0) {
```

Comment:
if count < 2, no need for else as you can return inside the if

Reply:
Done.